### PR TITLE
Add requirements for Prometheus connector

### PR DESCRIPTION
**Pull request by v-jizhang**
Cherry-pick of https://github.com/trinodb/trino/pull/8424
Add a standardized requirements to the Prometheus connector page.

Co-authored-by: Joe Lodin <joe.lodin@starburstdata.com>

```
== NO RELEASE NOTES ==

```

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/prometheus.rst [hunk 1]
```diff
@@ -10,6 +10,15 @@ In this case the ``up`` metric is taken from the Presto query table name. ``21d`
 corresponds to the ``timestamp`` field. Presto queries are translated from their use of the ``timestamp`` field to a duration and time value
 as needed. Presto splits are generated by dividing the query range into attempted equal chunks.
 
+Requirements
+------------
+
+To query Prometheus, you need:
+
+* Network access from the Presto coordinator and workers to the Prometheus
+  server. The default port is 9090.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/connector/prometheus.rst:19]:
```
``9090``
```

**Review by mayankgarg1990 — CHANGES_REQUESTED** [review]
You don't need any release notes for this PR . You can just say - 

```
== NO RELEASE NOTES ==
```

**Comment by v-jizhang** [comment]
> You don't need any release notes for this PR . You can just say -
> 
> ```
> == NO RELEASE NOTES ==
> ```

Fixed. Thanks

File: presto-docs/src/main/sphinx/connector/prometheus.rst (hunk 1 above)
Comment (v-jizhang) [reply presto-docs/src/main/sphinx/connector/prometheus.rst]: Fixed. Thanks!

**Review by mayankgarg1990 — APPROVED** [review]

**Review by steveburnett — CHANGES_REQUESTED** [review]
Local doc build failed: 

ImportError: cannot import name 'environmentfilter' from 'jinja2' (/Users/steveburnett/Documents/GitHub/presto/presto-docs/presto-docs-venv/lib/python3.9/site-packages/jinja2/__init__.py)
make: *** [html] Error 1

git merge master, then the doc build ran successfully. 

Reviewed local doc build and the doc looks good. Update the PR and re-request my review.

**Comment by steveburnett** [comment]
Closing due to lack of activity and age. If this is still relevant, please re-open this PR, rebase, and resolve any file conflicts.

**Pull request closed without merging**